lib/ansible/config/base.yml

```diff
@@ -1708,4 +1708,18 @@ NETCONF_SSH_CONFIG:
   - {key: ssh_config, section: netconf_connection}
   yaml: {key: netconf_connection.ssh_config}
   default: null
+STRING_CONVERSION_ACTION:
+  version_added: '2.8'
+  description:
+    - Action to take when a parameter value is converted to a string. For string fields, values such as '1.00',
```

Comment:
you might want to specify this is for 'module options' and does not appy to other variable conversions